### PR TITLE
Proposal

### PR DESCRIPTION
**Pull request by dmadisetti**
Please review all (hoping this gets picked up by the slack bot)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,6 @@
-# Super cool project name
+# Using Unsupervsed Learned to provide a Competive Edge in a Classification Competition
```
Comment (tding1) [inline README.md:1]: - some typos here: "Using unsupervised learning", and "competitive"
- i am not so sure what does "a competitive edge" mean?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,6 @@
...
- - Look into segmentation problems
-   - Brains
-   - robots
-
-Tianyu
-  - Generalized neural Models
-
-Dylan 
-  - General board games
+This project seeks to use SOTA unsupervised techniques to develop a competive
```
Comment (tding1) [inline README.md:4]: same typo here "competitive"

File: proposal/report_group_XX.tex [hunk 3]
```diff
@@ -8,31 +8,109 @@
 \newcommand{\todo}[1]{\textcolor{red}{#1}}
 
 \title{CS482/682 Final Project Report Group \todo{XX}\\
-	\large \todo{your project title here}}
+  \large Using Unsupervsed Learned to provide a Competive Edge in a Classification Competition}
```
Comment (tding1) [inline proposal/report_group_XX.tex]: typos like before

File: proposal/report_group_XX.tex [hunk 4]
```diff
@@ -8,31 +8,109 @@
...
+  \section{Introduction}
+
+  As a case study in how unsupervised learning can be used to tackle
+  established, benchmarkable problems, this project seeks to compete in the
+  \href{https://www.kaggle.com/c/pku-autonomous-driving}{Peking University/Baidu
+  Autonomous Vehicle Challenge}. This challenge provides pictures of traffic
+  while driving, and requires a prediction of the captured cars model type, and
+  positioning (yaw, pitch, roll, x, y, z). Auxiliary data is provided in the
+  form of 3D models for the cars of interest, and segmentation masks of the cars
+  against the background.
```
Comment (tding1) [inline proposal/report_group_XX.tex]: - This corresponds to "the brief summary of the dataset".
- better to be a separated section?

File: proposal/report_group_XX.tex [hunk 5]
```diff
@@ -8,31 +8,109 @@
...
-	\section{Discussion} \todo{make sure you use this section to showcase your understanding of the results you achieved and whether you could have / should have done things differently }
-
-\end{document}
\ No newline at end of file
+
+  \maketitle
+
+  % do not write an abstract
+  \section{Introduction}
+
```
Comment (tding1) [inline proposal/report_group_XX.tex]: - first of all, we should have a brief problem statement, maybe we can paraphrase the description from the kaggle description?

File: proposal/report_group_XX.tex [hunk 6]
```diff
@@ -8,31 +8,109 @@
...
+  As a case study in how unsupervised learning can be used to tackle
+  established, benchmarkable problems, this project seeks to compete in the
+  \href{https://www.kaggle.com/c/pku-autonomous-driving}{Peking University/Baidu
+  Autonomous Vehicle Challenge}. This challenge provides pictures of traffic
+  while driving, and requires a prediction of the captured cars model type, and
+  positioning (yaw, pitch, roll, x, y, z). Auxiliary data is provided in the
+  form of 3D models for the cars of interest, and segmentation masks of the cars
+  against the background.
+
```
Comment (tding1) [inline proposal/report_group_XX.tex]: - The following part is "the outline of how we attempt to solve the problem"

File: proposal/report_group_XX.tex [hunk 7]
```diff
@@ -8,31 +8,109 @@
...
+  understood that the competition task is non-trivial- thus this approach will
+  guarantee some concrete deliverables.
+
+  \textbf{Stage 1}. In the first stage, unsupervised segmentation of the
+  provided cars and their backgrounds. Unsupervised segmentation is well studied
+  as shown by \todo{find some refs, add some more detail\ref{}}. This methodology can be benchmarked
+  against the provided segmentation masks.  Completion of this stage will allow
+  for car segmentation from an arbitrary image. Further work may include
+  benchmarking this unsupervised approach to a supervised approach (using the
+  provided segmentation masks as ground truth).
```
Comment (sundeyichina) [inline proposal/report_group_XX.tex]: No comments right now

**Review by sundeyichina — COMMENTED** [review]
Editing in person with @dmadisetti

**Pull request merged**